wgpu-types/src/lib.rs

```diff
@@ -5196,11 +5196,16 @@ pub struct RenderBundleDepthStencil {
     ///
     /// This must match the [`RenderPassDepthStencilAttachment::depth_ops`] of the renderpass this render bundle is executed in.
     /// If depth_ops is `Some(..)` this must be false. If it is `None` this must be true.
+    ///
+    /// [`RenderPassDepthStencilAttachment::depth_ops`]: ../wgpu/struct.RenderPassDepthStencilAttachment.html#structfield.depth_ops
```

Comment:
Will this also work locally?

Reply:
You mean, when viewing docs using `file:` URLs? Yes.

It also works in both the `wgpu_types` and `wgpu` occurrences of the `RenderBundleDepthStencil` page.